eclipse-che/che-operator
# 2nd proofreading of api/v2/checluster_types.go (grammar, interpunction, clarity etc.)

### What does this PR do?
2nd proofreading of api/v2/checluster_types.go (grammar, interpunction, clarity etc.)


### Screenshot/screencast of this PR

![image](https://user-images.githubusercontent.com/80100533/173787746-a2197d49-9ab6-4e8a-984b-4945494dc8c5.png)


### What issues does this PR fix or reference?
 
https://issues.redhat.com/browse/RHDEVDOCS-4053


### How to test this PR?

please check edited content

```bash
cat EOF << > /tmp/patch.yaml
<PATCH_CONTENT>
EOF

chectl server:deploy \
     --installer operator \
     --platform <PLATFORM_TO_DEPLOY> \
     --che-operator-image <CUSTOM_OPERATOR_IMAGE> \
     --che-operator-cr-patch-yaml /tmp/patch.yaml
```

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
